sql/moz-fx-data-shared-prod/google_ads_derived/android_app_campaign_stats_v1/checks.sql

```diff
@@ -1,2 +1,2 @@
 #fail
-{{ is_unique(["date", "campaign", "ad_group"], "date = DATE_SUB(@submission_date, INTERVAL 27 DAY)") }}
+{{ is_unique(["date", "campaign", "ad_group"]) }}
```

Comment:
Apologies, I just thought about it again and this is fine. It just means that the check will be executed against the whole table and not just a single partition. In this case I believe the table is quite small so it doesn't really matter.